easybuild/easyblocks/f/ferret.py

```diff
@@ -48,80 +53,156 @@ def configure_step(self):
         """Configure Ferret build."""
 
         buildtype = "x86_64-linux"
-
-        try:
-            os.chdir('FERRET')
-        except OSError, err:
-            raise EasyBuildError("Failed to change to FERRET dir: %s", err)
+        if LooseVersion(self.version) < LooseVersion("7.3"):
+            try:
+                os.chdir('FERRET')
+            except OSError, err:
+                raise EasyBuildError("Failed to change to FERRET dir: %s", err)
 
         deps = ['HDF5', 'netCDF', 'Java']
 
         for name in deps:
             if not get_software_root(name):
                 raise EasyBuildError("%s module not loaded?", name)
 
-        fn = "site_specific.mk"
+        if LooseVersion(self.version) >= LooseVersion("7.3"):
+            try:
+                shutil.copy2('external_functions/ef_utility/site_specific.mk.in',
+                             'external_functions/ef_utility/site_specific.mk')
+                shutil.copy2('site_specific.mk.in', 'site_specific.mk')
```

Comment:
please use the `copy_file` function from `filetools` (no need for a `try`/`except` around it)